federated_learning/financial_fraud_detection/README.md

```diff
@@ -80,6 +87,21 @@ cd tree-based/
 nvflare simulator jobs/creditcard_5_bagging_IID_split_uniform_lr -w ${PWD}/workspaces/xgboost_workspace_5_bagging_exponential_split_scaled_lr -n 5 -t 5
 ```
 
+You will see the execution as shown in the picture
+![execution](./figures/execution.png)
+Each Client loads the global model and does the local training, gets the train-auc and valid-auc.
+After the federated learning phase, the server will store the global model.
+![execution](./figures/store_model.png)
```

Comment:
I tried to adjust the sentences to make it flows better. Can you confirm the sentence is correct? Thanks.

```suggestion

Each client loads the global model and starts local training. The application will periodically print the train-auc (AUC calculated from the training dataset) and valid-auc (AUC calculated from the validation dataset).

After the federated learning phase finishes, the server will store the global model.

![execution](./figures/store_model.png)
```

Reply:
Thanks a lot, This version is more fluent.